crates/bevy_pbr/src/material_bind_groups.rs

```diff
@@ -857,6 +915,10 @@ where
                     }
                 }
 
+                OwnedBindingResource::Data(_) => {
+                    // The size of a data buffer is unlimited.
```

Comment:
Storage and uniform buffers have maximum sizes, so perhaps that is the limit here. Uniform buffer limits are likely to be hit in some cases, but storage buffer ones practically not.

Reply:
Ah but this is only for bindless? Hrm. I was hoping flat buffers would be done for all cases. Do you see a way to do that? And also support uniform buffers?

Reply:
Non-bindless support wouldn't be useful until the non-bindless allocator has the ability to merge allocations that have identical resources other than buffers, like the bindless allocator can. In order to not have allocation be O(n), this would probably require implementing some kind of hashing scheme. So I'd like to have non-bindless support, but it'd probably be better for its own PR so as not to have too much unrelated stuff in this one.